### DISCUSSION TITLE
Time and Freq Synchronization

### DISCUSSION THREAD

**Discussion by cacent96** [NVlabs/sionna#80] [Q&A] [answered]
Hello,

Based from the Sionna API module, I notice there is no synchronization module specify in Sionna. In my case, I would required the time and frequency synchronization after the matched filter + downsampling. Any suggestion on this? 

Thank you.

**Comment by jhoydis** [comment, marked as answer]
Correct, there is no such module for the moment. You would need to implement it yourself. The precise algorithms to use depend on the structure of the transmitted signal, e.g., type and location of the synchronisation  sequence.

**Reply by cacent96** [reply]
Any plan on including this module into Sionna?

**Reply by jhoydis** [reply]
It is currently not prioritised on our feature list.
